qcodes/instrument_drivers/QDev/QDac.py

```diff
@@ -30,7 +31,8 @@ class QDac(VisaInstrument):
 
     # set nonzero value (seconds) to accept older status when reading settings
     max_status_age = 1
-
+    # this driver will be deprecated in favor of QDac_channels.py
+    @deprecate
```

Comment:
please use `reason` and `alternative` arguments of this decorator to provide our users with reason and alternative. after that, the comment `this driver will be deprecated in favor of QDac_channels.py` will be redundant. (Also note that it is extremely unlikely that the users will read that comment.)